src/components/viewpointPage/Outliner.jsx

```diff
@@ -265,6 +272,15 @@ class Outliner extends React.Component {
     return this.changing;
   }
 
+  async applyDelete() {
+    let db = new Hypertopic((await conf).services);
+    await db.get({_id: this.props.match.params.id})
+      .then(db.delete)
+      .then(_log)
```

Comment:
I suppose you needed logging when you developed the feature. But now, you can remove this line.